.github/PULL_REQUEST_TEMPLATE/release_template.md

```diff
@@ -8,11 +8,10 @@ TODO
 
 ## TODOs
 
-- [ ] test locally
-- [ ] upload assets to draft release
-- [ ] test one of the release packages locally
+Follow "Publishing a release" steps in `ci/README.md`
+
+<!-- Note some of these steps below are redudant since they're listed in the "Publishing a release" docs -->
+
 - [ ] double-check github release tag is the commit with artifacts (_note gets messed up after uploading assets_)
```

Comment:
Specifically wrt L15 - this is no longer the case after we moved from `hub` to `gh`, so we can remove that line.

Reply:
Oh awesome! I'll remove that here and anywhere else it's referenced.

Reply:
Fixed!